integr8ly/integreatly-operator
# INTLY-10512 Remove unused Keycloak dashboard

# Description
The keycloak metrics dashboard was renamed from keycloak to keycloak-metrics in v12, This PR removes the old unused dashboard from existing installs

## Verification
To verify: 
1. Install RHMI from 2.7.2 branch, 
2. Verify Keycloak dashboard exists in RHSSO and UserSSO grafana folders. 
3. (Optional) Install 2.8.0 branch and verify keycloak and keycloak-metrics dashboards exist
4. Install from this branch
5. Verify Keycloak dashboard is removed and keycloak-metrics dashboard remains. 

## Type of change



- [ X] Bug fix (non-breaking change which fixes an issue)
